tests/buttons.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { expect, test } from '@playwright/test'
+
+test('test Buttons', async ({ page }) => {
+	await page.goto('/');
+	await page.locator('div').filter({ hasText: /^Elements$/ }).first().click()
+	await page.locator('//span[text()="Buttons"]').click()
+	await page.locator('#doubleClickBtn').click()
+	await page.locator('#rightClickBtn').click()
```

Comment:
Here you should use **Right**  click, not usual click, and add assertion that you can see corresponding text

Reply:
done